cmd/vault.go

```diff
@@ -283,6 +282,17 @@ func CreateVault(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func verifyVaultDoesNotExist(vaultname string) error {
+	v, err := DefaultAPIClient.GetVault(vaultname)
+	if err != nil && err.Error() != client.ErrorVaultNotFound {
+		return errors.Errorf("error when checking vault %s: %v \nmaybe the vault already exists", vaultname, err)
```

Comment:
Meldingen "maybe the vault already exists" er misvisende.

Reply:
Nei, det mener jeg den ikke er.  Det kan være andre feil årsaker, f.eks. serverfeil, nettverksfeil, nedetid etc.

Reply:
Men hvorfor skal ao anta noe om årsaken så lenge det kommer en eksakt feilmelding først.

Eksempel: 

_error when checking vault demo: Token has expired for (http://boober-aurora.utv.paas.skead.no). Please login: ao login <affiliation> 
maybe the vault already exists_

og:

_error when checking vault bar: Forbidden: You (m87950) do not have required permissions to operate on this vault. You have [APP_AU_utv, APP_AppD_Read_Only_Prod, APP_INMO_utv, APP_MAP_utv, APP_PaaS_utv, UTV_Utvikler]. 
maybe the vault already exists_

Reply:
Jeg tenkte ikke å si at dette er årsaken, men å indikere at man i en del situasjoner må finne et annet navn på vaultet.

Ta denne feilmeldingen: 
_error when checking vault bar: Forbidden: You (m87950) do not have required permissions to operate on this vault. You have [APP_AU_utv, APP_AppD_Read_Only_Prod, APP_INMO_utv, APP_MAP_utv, APP_PaaS_utv, UTV_Utvikler]._

Den er helt korrekt, men den kan få brukeren til å tro at det handler om feil rettigheter.  Jeg forsøkte med "maybe the vault already exists" å hinte om at det er dette som sannsynligvis er problemet.

Jeg ser likevel at i en del tilfeller er det ikke sannsynlig.  "maybe" indikerer nettopp at vi ikke vet om vaultet eksisterer, men jeg ser at det for noen feilmeldinger er irrelevant.

Jeg fjerner den, så får vi håpe at folk er oppegående og skjønner at man må velge et annet navn, selv om feilmeldingen handler om permissions.